### ISSUE TITLE
How to compare distance between two features?

### ISSUE THREAD

**Issue by prnvjb** [sirius-ai/MobileFaceNet_TF#35]
Hi,
How to compare distance/similarity between two faces' features.
I tries below methods but failed.
diff = np.subtract(f2, f1)
dist = np.sum(np.square(diff), 1)
dist = np.linalg.norm(diff, axis=1)
dist = sklearn.metrics.pairwise_distances(f1,f2, metric='cosine')

Thanks for the help!!

**Comment by sirius-ai** [comment]
if use euclidean distance:
dist = np.sqrt(np.sum(np.square(np.subtract(f2, f1))))

**Comment by sirius-ai** [comment]
close, when need reopen.
